### PR TITLE
Issue #87

### PR DESCRIPTION
**Pull request by NiklasHogefjord**
* Tweak in checkout order creation flow. When order is created in WooCommerce we do not listen to a hashchange anymore. Instead the response is returned entirely in the ajax function  that originally submitted the WooCommerce form. This solves a compatibility issue we have seen with some GTM/Analytics plugins.
* Adds stacktrace in log messages.

### CODE REVIEW

File: src/assets/js/bco-checkout.js [hunk 1]
```diff
@@ -510,6 +485,73 @@ jQuery(function($) {
...
+				url: bco_wc_params.submit_order,
+				data: $('form.checkout').serialize(),
+				dataType: 'json',
+				success: function( data ) {
+					try {
+						if ( 'success' === data.result ) {
+							bco_wc.logToFile( 'Successfully placed order. Sending bco_order_validation true to Billmate' );
+
+							$( 'body' ).trigger( 'bco_order_validation', true );
+							console.log('data.redirect_url');
```
Comment (MattiasNilsson) [inline src/assets/js/bco-checkout.js:510]: Remove console.log

File: src/assets/js/bco-checkout.js [hunk 2]
```diff
@@ -510,6 +485,73 @@ jQuery(function($) {
...
+							console.log(data.redirect_url);
```
Comment (MattiasNilsson) [inline src/assets/js/bco-checkout.js:511]: Remove console.log

**Review by MattiasNilsson — CHANGES_REQUESTED** [review]
This looks good, just some minor nitpicks with console.log

File: src/assets/js/bco-checkout.js (hunk 1 above)
Comment (NiklasHogefjord) [reply src/assets/js/bco-checkout.js:510]: Thanks @MattiasNilsson. Do you want it removed even though we remove all console.log in the minified version of the file?

Comment (MattiasNilsson) [reply src/assets/js/bco-checkout.js:510]: Yes instead of keeping it, I mean the code it just there for debugging. So remove :)

Comment (NiklasHogefjord) [reply src/assets/js/bco-checkout.js]: OK, thanks :)

**Pull request merged**